pyof/v0x01/controller2switch/stats_reply.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, xid=None, body_type=None, flags=None, body=b''):
         super().__init__(xid)
         self.body_type = body_type
         self.flags = flags
-        self.body = body
+        self.body = BinaryData() if body is None else body
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.